Dockerfile

```diff
@@ -2,18 +2,18 @@ ARG ARCH=amd64
 
 # The node version here should match the version of the runtime image which is
 # specified in the base-image subdirectory in the project
-FROM resin/rpi-node:6.13.1-slim as rpi-node-base
-FROM resin/armv7hf-node:6.13.1-slim as armv7hf-node-base
-FROM resin/aarch64-node:6.13.1-slim as aarch64-node-base
+FROM balenalib/rpi-node:6.16.0 as rpi-node-base
+FROM balenalib/armv7hf-node:6.16.0 as armv7hf-node-base
```

Comment:
I think this will be fine, but this *could* cause issues given the build and runtime will be using slightly different nodejs versions. I'd strongly recommend either sticking to 6.13.1 here (and creating balenalib/ARCH-node builds) or moving balena-supervisor-base to 6.16.0.

Reply:
Yeah I think this is fine with just a minor version, but it's worth moving the base too. The only problem there is the base is on v8, and that causes big issues with deltas, so we can't do that. Getting the other base images generated is probably the easiest.